### PR TITLE
adds database package and local database setup

### PR DESCRIPTION
**Pull request by patimejia**
1. created a new yarn workspace called `packages/database`
2. created `README.md` to show how to install supabase
3. created `.env` to hold the database connection string
4. created migration and `seed.sql` files to initialize the database with data
5. created `index.js` to show an example of how to query the database

Screenshot 2023-04-01 at 7 44 21 PM

Screenshot 2023-04-01 at 7 44 59 PM

### CODE REVIEW

**Comment by patimejia** [comment]
Hi @mattkuznicki-ll,

Just wanted to drop a quick message to let you know that I renamed the `backend` folder to `database`. However, I was revisiting the documentation and had a thought. Would you prefer if we put the `database` folder inside a `backend folder`, e.g. a new `backend/database` structure? If so, I can make the change. Thanks for your input - it's always appreciated.

File: packages/database/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { createClient } from "@supabase/supabase-js";
+import * as dotenv from "dotenv"; // see https://github.com/motdotla/dotenv#how-do-i-use-dotenv-with-import
+dotenv.config();
+
+const supabase = createClient(
+  process.env.SUPABASE_URL,
```
Comment (mattkuznicki-ll) [inline packages/database/index.js:6]: Thank you for not hardcoding these values! 👍

**Comment by mattkuznicki-ll** [comment]
Hi @patimejia I'd really not recommend renaming the folder. Remember that the database and the backend are not the same - the backend will use the database & the database is called by the backend. So the packages/database folder is probably a better idea than removing the backend folder.
It looks like the use of supabase would introduce a Docker dependency for backend contributors. Is there any way not to have to introduce Docker into this? I'm hoping we can keep the amount of tooling contributors need to set up down if we can.

**Review by mattkuznicki-ll — COMMENTED** [review]
Thanks for this PR @patimejia !

Right now, we're thinking we'll be doing with an Azure - based database for production and for any remote staging environment, so Supabase would just be used for contributors who are locally developing. Does Supabase offer much advantage for local development over a MySQL installation?

**Comment by patimejia** [comment]
Reverted branch to local Supabase implementation, keep as example, move forward with project and new structures/tech. Closes issue #9. Thanks!

**Comment by pid1** [comment]
@mattkuznicki-ll This looks good to me, can this be merged?

**Comment by mattkuznicki-ll** [comment]
> @mattkuznicki-ll This looks good to me, can this be merged?

@pid1 @patimejia with the consensus looking like we're splitting away from a monorepo and to a separate database service in a separate repository (to be used by Bechamel, the API that the front-end components will talk to), the database looks like it's not going to live in this repository. Also suspect that the local use database will evolve to the Azure database service emulator but that's less of an issue.
There's great work in here and I don't want to lose it, but my recommendation is that we hold this & hold database creation in general for a bit, and close this PR in this repo. May make more sense to get more done on the API side and something usable in mock or other form there before starting into the database componentry.
Thoughts?

**Pull request closed without merging**